sci-libs/atlas/atlas-3.10.3.recipe

```diff
@@ -0,0 +1,145 @@
+SUMMARY="Automatically Tuned Linear Algebra Software"
+DESCRIPTION="A software library for linear algebra. It provides a mature open \
+source implementation of BLAS APIs for C and Fortran77."
+HOMEPAGE="http://math-atlas.sourceforge.net/"
+COPYRIGHT="2011-2016 The ATLAS Project"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://downloads.sf.net/math-atlas/atlas$portVersion.tar.bz2"
+CHECKSUM_SHA256="2688eb733a6c5f78a18ef32144039adcd62fabce66f2eb51dd59dde806a6d2b7"
+SOURCE_DIR="ATLAS"
+PATCHES="atlas-$portVersion.patchset"
+ADDITIONAL_FILES="makefile.shared.mt
+	makefile.shared.st"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	atlas$secondaryArchSuffix = $portVersion
+	lib:libatlas$secondaryArchSuffix = $portVersion compat >=3
+	lib:libcblas$secondaryArchSuffix = $portVersion compat >=3
+	lib:libf77blas$secondaryArchSuffix = $portVersion compat >=3
+	lib:liblapack$secondaryArchSuffix = $portVersion compat >=3
+	lib:libptcblas$secondaryArchSuffix = $portVersion compat >=3
+	lib:libptf77blas$secondaryArchSuffix = $portVersion compat >=3
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgfortran$secondaryArchSuffix
+	lib:libgomp$secondaryArchSuffix
+	"
+CONFLICTS="
+	blas$secondaryArchSuffix
+	cblas$secondaryArchSuffix
+	lapack$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	atlas${secondaryArchSuffix}_devel = $portVersion compat >=3
```

Comment:
For **`atlas${secondaryArchSuffix}_devel`** there should be no **`compat`** at all.